### DISCUSSION TITLE
Hot Topic: Telescopic Batons and the Balancing Behind it

### DISCUSSION THREAD

**Discussion by challade** [shiptest-ss13/Shiptest#813] [General]
People vote their dreams.

Recently there have been a number of complaints on a certain item that captains of a ship spawn with, the telescopic baton. This baton, is a small object that when deployed appear on your sprite and will stun anyone down for a period of time. You can chain this and keep someone down. In order to make it work you must walk up to someone and click on them with it. it's more or less uncommon, only the lead of a ship gets one per ship.

The complaint is this: 

It's unfair. You get chain locked into a stun because someone walked up to you and clicked you, effectively losing the engagement. It feels unfair that if you have a gun, someone managed to walk up to you and chain lock you, effectively stripping and or killing you. All melee 1v1s will be lost to the guy who can land the baton unless you have a shield, and if you run out of ammo and you don't have a baton either, you're done. Instant hard stuns are bad, that's why Tazers were removed.

Counter argument: 

It is unfair. It's supposed to be unfair, an equalizer if the enemy has an entire armory on them. Otherwise, one guy with 5 guns just attritions you, and if you walk up to him and DON'T get a stun, he'll use it as a chance to unload on you. That's assuming you reach him in the first place. Two shots of most projectiles, with armor, will slow you down already, not to mention no armor. If you manage to dodge everything AND reach the guy, you have to stun or they'll just unload on you. Zig zagging doesn't really work on ships when they're 1 to 2 tile wide halls. If you let a guy run you down whilst having an armory on you, it's your fault. You can also be in possession of a baton too. If you do run out of ammo and it's down to two people with batons, it's a skill issue. You missed everything and now you have to see who clicks who first with the baton. It's also wrong to compare them to tazers since tazers are ranged. Ranged is already better, Batons are not only rare, they're the equalizer between one Captain and some guy who has 5 guns. Not to mention that the baton has a cooldown, meaning you can't chain stun multiple people, only one person. Again, gotta get in melee range first. 

Personally, I'm biased in favor of the Baton, if you nerf it you lose it, it'll be absolutely worthless against anyone with a gun, which it already has a melee handicap, so good luck getting to one. Plus it's rare, and two people with it becomes a skill contest. If you really want to deal with it, become an unstunnable golem. If we do nerf it, extend the CD so you can only chain stun one person at a time.

**Comment by Apogee-dev** [comment]
The baton should be kept as is. Yes, it's an instant hard stun. That is the point. If you're loaded down with an entire armory and you let someone walk into melee range and click on you it's entirely your fault, and nerfing it defeats the entire point of the baton- it's supposed to be a non-lethal disable against poorly to moderately armed opponents that will likely outnumber the person using the baton (this applies to stun batons as well as telebatons, and the classic example would be a mob of greytiders besieging the brig or the bridge on a typical station). If you have a ton of guns and you see a guy running at you with a baton, _just shoot him._

**Comment by Kapu1178** [comment]
just saying, its not a hardstun. its a knockdown.

**Reply by Ms-Mee** [reply]
A 1.5 seconds knockdown on a 4 seconds cooldown, in fact
